plugins/code-coverage/README.md

```diff
@@ -2,6 +2,10 @@
 
 This is the frontend part of the code-coverage plugin. It displays code coverage summaries for your entities.
 
+## Installation
+
+`yarn workspace app add @backstage/plugin-code-coverage`
+
```

Comment:
there should be a couple of lines about adding the actual view, e.g. like we do in the example app (remember to include an `import` line too so people see where it's at)

https://github.com/backstage/backstage/blob/0216863048d8b981fa1cbcbe86300f8e7c6c4011/packages/app/src/components/catalog/EntityPage.tsx#L480